core/src/main/scala/com/evolutiongaming/catshelper/FeatureToggled.scala

```diff
@@ -75,7 +75,7 @@ object FeatureToggled {
 
       featureToggledResource = {
         val toggle = (v: Boolean) => toggleStateOf(v).flatMap { ts =>
-          flagRef.getAndSet(ts).flatMap(_.next.complete(ts)).uncancelable
+          Concurrent[F].uncancelable(_ => flagRef.getAndSet(ts).flatMap(_.next.complete(ts).void))          
```

Comment:
could you explain more about this construction?

Reply:
it's basically the same, just changed with the introduction of Poll:  
https://typelevel.org/cats-effect/docs/migration-guide#bracket

Reply:
Actually, there's a syntax in MonadCancel extension, will use that

Reply:
done